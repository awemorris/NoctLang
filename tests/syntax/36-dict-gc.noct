func main() {
    var dict = {
        var001: 0,
        var002: 1,
        var003: 2,
        var004: 3,
        var005: 4,
        var006: 5,
        var007: 6,
        var008: 7,
        var009: 8,
        var010: 9,
        var011: 10,
        var012: 11,
        var013: 12,
        var014: 13,
        var015: 14,
        var016: 15,
        var017: 16
    };

    fast_gc();

    dict.var018 = 17;
    dict.var019 = 18;
    dict.var020 = 19;
    dict.var021 = 20;
    dict.var022 = 21;
    dict.var023 = 22;
    dict.var024 = 23;
    dict.var025 = 24;
    dict.var026 = 25;
    dict.var027 = 26;
    dict.var028 = 27;
    dict.var029 = 28;
    dict.var030 = 29;
    dict.var031 = 30;
    dict.var032 = 31;
    dict.var033 = 32;

    for (k, v in dict)
        print(k + "=" + v);
}
