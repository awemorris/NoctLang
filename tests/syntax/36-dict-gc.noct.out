var023=22
var008=7
var010=9
var029=28
var003=2
var019=18
var020=19
var027=26
var004=3
var014=13
var030=29
var024=23
var009=8
var013=12
var033=32
var018=17
var021=20
var005=4
var017=16
var025=24
var006=5
var012=11
var032=31
var001=0
var022=21
var011=10
var028=27
var002=1
var016=15
var026=25
var007=6
var015=14
var031=30
